src/kOS/AddOns/InfernalRobotics/IRHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace kOS.AddOns.InfernalRobotics
+{
+    [KSPAddon(KSPAddon.Startup.EveryScene, false)]
+    public class IRHandler : MonoBehaviour
+    {
+        public void Start()
+        {
+            IRWrapper.InitWrapper();
+            if (IRWrapper.APIReady)
+            {
```

Comment:
Just left it here as a placeholder in case I wanted to add some event handling here later, but never needed it so far.